data_structures/bloom_filters/bloom_filter/bloom_filter.c

```diff
@@ -0,0 +1,146 @@
+#include "./bloom_filter.h"
+
+#include <math.h>
+
+static inline int murmur_32_scramble(int k)
```

Comment:
Did you copied this implementation from [Wikipedia](https://en.wikipedia.org/wiki/MurmurHash#Algorithm)? Not that's necessarily a bad thing, but if you did you should give credit where it's due